site/docs/components/system-status/accessibility.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Best practices
+
+- Use the aria-live="assertive" attribute for error messages to ensure they are announced immediately, interrupting any ongoing speech to alert the user to critical issues requiring immediate attention.
```

Comment:
We should update this to role="alert" and role "status"

Reply:
done